prow/metrics/metrics.go

```diff
@@ -31,20 +31,20 @@ import (
 	"k8s.io/test-infra/prow/interrupts"
 )
 
-const metricsPort = 9090
+const defaultMetricsPort = 9090
```

Comment:
Should we use this const everywhere instead of the few places where you still have 9090?
Same question for an hypothetical const for pprof-port (6060)?